GameReviewBackend/BusinessLogic/Infrastructure/IgdbApiService.cs

```diff
@@ -15,8 +15,9 @@ namespace BusinessLogic.Infrastructure
     public class IgdbApiService : IIgdbApiService
     {
         GenericRepository<DockerDbContext> _genericRepository;
-        private string CLIENT_ID = "0w2j0mktoxkvehox8hl42p15tladu0";
-        private string CLIENT_SECRET = "gpptkmravp51lkhl5d22msu8i2rs2f";
+        private readonly string CLIENT_ID = "0w2j0mktoxkvehox8hl42p15tladu0";
+        private readonly string CLIENT_SECRET = "gpptkmravp51lkhl5d22msu8i2rs2f";
```

Comment:
this secret should probably not be hardcoded

Reply:
removed these and replaced with config secrets